ang/civicase.ang.php

```diff
@@ -158,7 +158,7 @@ function set_case_actions(&$options, $caseCategoryPermissions) {
       'icon' => 'fa-trash',
     ];
   }
-  if (CRM_Core_Permission::check(CRM_Civicase_Service_ExportPermission::NAME)) {
+  if (CRM_Core_Permission::check(CRM_Civicase_Hook_Permissions_ExportCasesAndReports::PERMISSION_NAME)) {
```

Comment:
Can we do use `CRM_Civicase_Hook_Permissions_ExportCasesAndReports` as `ExportCasesAndReports` at the top of this file so we don't have this line as long